addon/components/frost-pick-list.js

```diff
@@ -1,6 +1,127 @@
 import Ember from 'ember'
-const {Component} = Ember
+const {A, Component, copy} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {sort} from 'ember-frost-sort'
+import layout from '../templates/components/frost-pick-list'
+import {PropTypes} from 'ember-prop-types'
 
 export default Component.extend({
-  classNames: ['frost-pick-list']
+  // == Dependencies ==========================================================
+
+  // == Properties ============================================================
+
+  classNames: ['frost-pick-list'],
+  selectedItems: A([]),
```

Comment:
Remove as is in `getDefaultProps()`.

Why aren't the rest of these properties in `getDefaultProps()` and are instead defined here?